### PR TITLE
look up player name based on ID #66

### PR DESCRIPTION
**Pull request by afeierman**
additionally, because there's now easy enough ways to move between player ID and name, it's not clear to me that PR #51 is needed

### CODE REVIEW

File: R/playername_lookup.R [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#' @examples
+#' \dontrun{playername_lookup(4885)}
+#' \dontrun{playername_lookup("kaaihki01")}
+
+playername_lookup <- function(id) {
+  if (!exists("chadwick_player_lu_table")) {
+    print("Be patient, this may take a few seconds...")
+    print("Data courtesy of the Chadwick Bureau Register (https://github.com/chadwickbureau/register)")
+    url <- "https://raw.githubusercontent.com/chadwickbureau/register/master/data/people.csv"
+    chadwick_player_lu_table <- read.csv(url)
```
Comment (BillPetti) [inline R/playername_lookup.R]: Can you switch this to `readr::read_csv()`? That will allow for the initial download of the Chadwick table to be quicker. I need to make this same change to the `playerid_lookup` function.

Comment (beanumber) [reply R/playername_lookup.R]: You saw that this change was made in #75 right?

Comment (BillPetti) [reply R/playername_lookup.R]: I missed it, but see it now.

Comment (afeierman) [reply R/playername_lookup.R]: Good catch-- agree with moving from read.csv to read_csv, I was just lazily copying from another function. Will change

**Comment by afeierman** [comment]
requested changes made in e59e859

**Pull request merged**